docs/installation/ssl.md

```diff
@@ -70,6 +70,38 @@ http {
 }
 ```
 
+Or an apache.conf file similar to:
+```virtual-site.conf
+<VirtualHost *:80>
+    ServerAdmin you@domain.com 
+    ServerName monica.yourdomain.com
+
+    RewriteEngine on
+    RewriteCond %{SERVER_NAME} =monica.yourdomain.com
+    # redirect all requests to port 80 to port 443 using 308 code
+    RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [END,QSA,NE,R=308]
+ </VirtualHost>
+```
+
+```virtual-site-ssl.conf
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+    ServerAdmin you@domain.com
+    ServerName monica.yourdomain.com
+
+    ProxyPreserveHost On
+    ProxyRequests Off
+    ProxyPass / http://localhost:3001/
+    ProxyPassReverse / http://localhost:3001/
```

Comment:
Maybe add
```
ProxyPassReverse / https://monica.yourdomain.com
```

Reply:
I don't think that will work. This is with docker.  monica.yourdomain.com resolves to the apache server, which forwards it to the docker container via localhost.

Reply:
OK you're right